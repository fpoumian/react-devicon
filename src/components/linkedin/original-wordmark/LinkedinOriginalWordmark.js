
import React from 'react'
import PropTypes from 'prop-types'
import SVGInline from 'react-svg-inline'
import iconSVG from './LinkedinOriginalWordmark.svg'
import { normalizeUnit } from '../../utils.js'

/** LinkedinOriginalWordmark */
function LinkedinOriginalWordmark({width, height}) {
  return (
     <SVGInline
      classSuffix="-devicon"
      className="LinkedinOriginalWordmark"
      svg={iconSVG}
      width={normalizeUnit(width)}
      height={normalizeUnit(height)}
    />
  )
}

LinkedinOriginalWordmark.propTypes = {
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
}

export default LinkedinOriginalWordmark
