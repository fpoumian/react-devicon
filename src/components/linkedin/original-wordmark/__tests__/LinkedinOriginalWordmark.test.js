import React from 'react'
import renderer from 'react-test-renderer'
import LinkedinOriginalWordmark from '../index.js'

it('renders correctly', () => {
  const tree = renderer.create(<LinkedinOriginalWordmark />).toJSON()
  expect(tree).toMatchSnapshot()
})
