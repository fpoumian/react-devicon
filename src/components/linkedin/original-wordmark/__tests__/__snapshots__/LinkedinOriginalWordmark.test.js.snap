// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="SVGInline LinkedinOriginalWordmark"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<svg class=\\"SVGInline-devicon LinkedinOriginalWordmark-devicon\\" id=\\"Linkedin\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 128 128\\"><defs><style>.cls-1{fill:#1d1d1b}.cls-2{fill:#0076b2}.cls-3{fill:#fff}</style></defs><title>Artboard 13</title><g id=\\"original-wordmark\\"><path class=\\"cls-1\\" d=\\"M.85 52.73h4.94v17.98h9.14v4.56H.85V52.73zM17.11 60h4.74v15.26h-4.74zm2.37-7.59a2.75 2.75 0 1 1-2.75 2.75 2.75 2.75 0 0 1 2.75-2.75M41.69 52.73h4.74V66.2l5.38-6.22h5.81l-6.22 7.07 6.09 8.22h-5.96l-5.04-7.55h-.06v7.55h-4.74V52.73zM24.48 60H29v2.09h.06a5 5 0 0 1 4.49-2.47c4.81 0 5.69 3.16 5.69 7.27v8.38h-4.7v-7.43c0-1.77 0-4-2.47-4s-2.85 1.93-2.85 3.92v7.55h-4.74z\\"/><path class=\\"cls-1\\" d=\\"M67.61 65.85a2.84 2.84 0 0 0-2.91-2.91 3.16 3.16 0 0 0-3.35 2.91zm4 6.77a8.35 8.35 0 0 1-6.48 3c-4.74 0-8.54-3.16-8.54-8.07s3.8-8.06 8.54-8.06c4.43 0 7.21 3.16 7.21 8.06v1.49h-11a3.54 3.54 0 0 0 3.57 3 4 4 0 0 0 3.38-1.87zM82.23 63.68A3.89 3.89 0 1 0 86 67.57a3.6 3.6 0 0 0-3.8-3.89m8.2 11.58H86v-2a6 6 0 0 1-4.71 2.4c-4.56 0-7.56-3.29-7.56-7.94 0-4.27 2.66-8.19 7-8.19a5.73 5.73 0 0 1 4.87 2h.06v-8.8h4.74z\\"/><path class=\\"cls-2\\" d=\\"M124.78 48H97.51a2.34 2.34 0 0 0-2.36 2.31v27.38A2.34 2.34 0 0 0 97.51 80h27.27a2.34 2.34 0 0 0 2.37-2.31V50.31a2.34 2.34 0 0 0-2.37-2.31z\\"/><path class=\\"cls-3\\" d=\\"M99.89 60h4.75v15.27h-4.75zm2.37-7.59a2.75 2.75 0 1 1-2.75 2.75 2.75 2.75 0 0 1 2.75-2.75M107.61 60h4.55v2.09h.06a5 5 0 0 1 4.49-2.47c4.81 0 5.69 3.16 5.69 7.27v8.38h-4.74v-7.43c0-1.77 0-4-2.47-4s-2.85 1.93-2.85 3.92v7.55h-4.74z\\"/></g></svg>",
    }
  }
/>
`;
